modules/configure-proxy-during-installation.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * configuration/configure-proxy.adoc
+:_module-type: PROCEDURE
+[id="configure-proxy-during-installation_{context}"]
+= Configure proxy during installation
+
+When you are installing {product-title} by using the `roxctl` command-line interface (CLI) or Helm, you can specify your proxy configuration during the installation.
+
+When you run the installer by using the `roxctl central generate` command, the installer generates the secrets and deployment configuration files for your environment. You can configure a proxy by editing the generated configuration secret (YAML) file. Currently, you cannot configure proxies by using the `roxctl` CLI. The configuration is stored in a Kubernetes secret and it is shared by both Central and Scanner.
+
+.Procedure
+
+. Open the configuration file `central/proxy-config-secret.yaml` from your deployment bundle directory.
++
+[NOTE]
+====
+If you are using `helm` the configuration file is at `central/templates/proxy-config-secret.yaml`.
+====
+. Edit the fields you want to modify in the configuration file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: stackrox
+  name: proxy-config
+type: Opaque
+stringData:
+  config.yaml: |-
+    # # NOTE: Both central and scanner should be restarted if this secret is changed.
+    # # While it is possible that some components will pick up the new proxy configuration
+    # # without a restart, it cannot be guaranteed that this will apply to every possible
+    # # integration etc.
+    # url: http://proxy.name:port
+    # username: username
+    # password: password
+    # # If the following value is set to true, the proxy wil NOT be excluded for the default hosts:
+    # # - *.stackrox, *.stackrox.svc
+    # # - localhost, localhost.localdomain, 127.0.0.0/8, ::1
+    # # - *.local
+    # omitDefaultExcludes: false
+    # excludes:  # hostnames (may include * components) for which not to use a proxy, like in-cluster repositories.
+    # - some.domain
+    # # The following configuration sections allow specifying a different proxy to be used for HTTP(S) connections.
+    # # If they are omitted, the above configuration is used for HTTP(S) connections as well as TCP connections.
+    # # If only the `http` section is given, it will be used for HTTPS connections as well.
+    # # Note: in most cases, a single, global proxy configuration is sufficient.
+    # http:
+    #   url: http://http-proxy.name:port
+    #   username: username
+    #   password: password
+    # https:
+    #   url: http://https-proxy.name:port
+    #   username: username
+    #   password: password
+----
+**  Adding a `username` and a `password` is optional, both at the beginning and in the `http` and `https` sections.
```

Comment:
Did you consider using callouts for these (or at least for the first few that apply to specific lines)? It might be easier to match up where in the above code block you're talking about